vsg/rules/align_tokens_in_region_between_tokens_unless_between_tokens.py

```diff
@@ -142,6 +145,19 @@ def analyze(self, oFile):
                            dAnalysis = {}
 
                    if self.case_control_statements_ends_group:
```

Comment:
We need a function to replace the condition in the if statement.  Maybe something like `is_case_control_enabled`.
```python
147                if is_case_control_enabled(self.case_control_statement_ends_group):
```
The function has the single task of determining if the correct values are set in `self.case_control_statement_ends_group`.
Create the function at the end of the file:
```python
def is_case_control_enabled(config):
    if config == 'skip_whens':
        return True
    return config
```
There is probably a better name than `skip_whens`, but we can check for it before returning the config value.  This should allow us to define any number of exceptions to the original behavior while keeping backwards compatibility.

Reply:
Done